R/reporter-progress.R

```diff
@@ -53,6 +56,8 @@ ProgressReporter <- R6::R6Class("ProgressReporter",
       # Capture at init so not affected by test settings
       self$frames <- cli::get_spinner()$frames
       self$width <- cli::console_width()
+      self$unicode <- getOption("cli.unicode")
+      self$colour <- getOption("crayon.enabled")
```

Comment:
I think this does not work if you have unicode and/or color because they were auto-detected. `cli::is_utf8_output()` and `crayon::has_color()` are probably better.

Reply:
Oh good catch, thanks!